Source/santasyncservice/SNTSyncRuleDownload.m

```diff
@@ -45,7 +45,10 @@ - (BOOL)sync {
                                                         error = e;
                                                         dispatch_semaphore_signal(sema);
                                                       }];
-  dispatch_semaphore_wait(sema, dispatch_time(DISPATCH_TIME_NOW, 300 * NSEC_PER_SEC));
+  if (dispatch_semaphore_wait(sema, dispatch_time(DISPATCH_TIME_NOW, 300 * NSEC_PER_SEC))) {
+    LOGE(@"Failed to add rule(s) to database: timeout connecting to daemon");
```

Comment:
nit: `connecting to` should really be `sending rules to daemon` or something

Reply:
Done!